lib/gnat.ex

```diff
@@ -69,8 +69,9 @@ defmodule Gnat do
   end
   ```
   """
-  @spec sub(GenServer.server, pid(), String.t, keyword()) :: {:ok, non_neg_integer()}
-  def sub(pid, subscriber, topic, opts \\ []), do: GenServer.call(pid, {:sub, subscriber, topic, opts})
+  @spec sub(GenServer.server(), pid(), String.t(), keyword()) :: {:ok, non_neg_integer()}
+  def sub(pid, subscriber, topic, opts \\ []),
+    do: GenServer.call(pid, {:sub, subscriber, topic, opts})
```

Comment:
I'm not a huge fan of changes like this which is making a newline if line > 80 characters.

I'd say if its close, leave as a single line. Otherwise lets block it like
```elixir
def sub(pid, subscriber, topic, opts \\ [])
  GenServer.call(pid, {:sub, subscriber, topic, opts})
end